File: Ex25ListViewCustom/app/src/main/java/com/jake5113/ex25listviewcustom/Item.java
package com.jake5113.ex25listviewcustom;

public class Item {
    String name; // "전현무"
    String nation; // "대한민국"
    int imgId; // R.drawable.korea

    // 객체를 생성(new)할때 자동으로 실행되는 특별한 메소드 - constructor  생성자
    public Item(String name, String nation, int imgId){
        this.name = name;
        this.nation = nation;
        this.imgId = imgId;
    }
}

File: Ex92KotlinRecyclerView/app/src/main/java/com/jake5113/ex92kotlinrecyclerview/Item.kt
package com.jake5113.ex92kotlinrecyclerview

// 매개변수면서 멤버변수

// data class : 데이터만 저장하는 목적의 클래스
// 일반클래스와 다르게 자동으로 equals() 할 때 객체 주소를 비교하지 않고
// 멤버값들을 비교해주도록 설계된 클래스 / 주생성자의 멤버만 비교함.
data class Item constructor(var name: String, var msg: String, var imgId: Int)